bin/run_ci_dataset.sh

```diff
@@ -35,17 +35,19 @@ print_error() {
 
 usage() {
     print_error
-    print_error "Usage: $0 -d DATASET [-h]"
+    print_error "Usage: $0 -d DATASET [-g NUM] [-h]"
     print_error
     print_error "Specifc options:"
     print_error "   -d          Dataset name"
+    print_error "   -g          Generation number (int)"
```

Comment:
Maybe "Butler" or "Middleware" generation number would be clearer if more verbose?